### PR TITLE
Ashley/add loading to drop down button

### PR DESCRIPTION
**Pull request by AshleyKensington**

### CODE REVIEW

File: src/components/DropdownButton/DropdownButton.tsx [hunk 1]
```diff
@@ -52,15 +52,27 @@ const Wrapper = styled.div<WrapperProps>`
...
+  ${props => (props.small ? "font-size: 12px" : "")};
+  ${props => (props.small ? "height: 20px" : "")};
+  ${props => (props.small ? "line-height: 20px" : "")};
+  padding: ${props =>
+    props.small
+      ? props.loading
+        ? "1px 16px 0 16px"
+        : "1px 8px 0 16px"
+      : "0 8px 0 16px"};
+  /* padding-right: 5px; */
```
Comment (mitchjhill) [inline src/components/DropdownButton/DropdownButton.tsx]: Remove sneaky comment

Comment (AshleyKensington) [reply src/components/DropdownButton/DropdownButton.tsx]: removed

File: src/components/DropdownButton/DropdownButton.tsx [hunk 2]
```diff
@@ -138,11 +148,12 @@ class DropdownButton extends Component<Props, State> {
             onClick={this.handleClick}
             icon={!isOpen ? 'keyboard_arrow_down' : 'keyboard_arrow_up'}
             small={small}
+            loading={loading}
           >
             {text}
           </StyledButton>
           {isOpen && !disabled ? (
-            <ResultWrapper small={small} onClick={() => this.setState({ isOpen: false })}>
+            <ResultWrapper small={small} onClick={() => this.setState({ isOpen: true })}>
```
Comment (mitchjhill) [inline src/components/DropdownButton/DropdownButton.tsx]: Did you mean to change this?

Comment (AshleyKensington) [reply src/components/DropdownButton/DropdownButton.tsx]: Changed

**Pull request merged**